CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Check for presence of the dataclass decorator
   (Issue [#76](https://github.com/StockwatchDev/application_settings/issues/76))
+- Enable file inclusion for toml config files
+  (Issue [#67](https://github.com/StockwatchDev/application_settings/issues/67))
 
```

Comment:
One note on upcoming issues. Although this is a very necessary and nice feature the parameter file generation as mentioned here:
https://github.com/StockwatchDev/application_settings/issues/74

probably will be very difficult! If we want to generate similar files we'll have to keep track where they come from, meaning we'll probably have to use a very different system as we're doing currently

Reply:
With that issue I meant generating a file during installation, i.e., before anything has been used. Then, a single file will suffice, I think, the user can make the split afterwards. For settings though, I do see the problem that you mention. Let's postpone that and make a separate issue.